unified-runtime/scripts/benchmarks/main.py

```diff
@@ -155,6 +154,9 @@ def main(directory, additional_env_vars, save_name, compare_names, filter):
         else []
     )
 
+    if not options.ur_adapter == "cuda":
+        suites.append(ComputeBench(directory))
```

Comment:
Also, it just occurred to me, that we don't necessarily need to disable all of the Compute Benchmarks, just the ones that are UR/L0 specific.
All these [benchmarks](https://github.com/intel/llvm/blob/sycl/unified-runtime/scripts/benchmarks/benches/compute.py#L61) that are suffixed with _sycl should work with any adapter.

Reply:
I tried it, but it seems to have some failures, so I tried to keep it simple for now and increase the working benchmarks working for cuda in a later PR if that's okay. (It could be due to a dependency mistake that I have locally but will look on that next)

Reply:
Sure, sounds good. Thanks for checking.